docs/worker-node/install-singularity.md

```diff
@@ -203,8 +212,9 @@ the container image.
 It is not enabled by default but recommended because it is less
 vulnerable to security problems than the similar default `overlay`
 option.
-In addition, the `overlay` option does not work on RHEL6 and does not
-work correctly on RHEL7 when container images are distributed by CVMFS.
+In addition, the `overlay` option does not work on RHEL6, does not
```

Comment:
Is there _ever_ a reason to use `overlay`? Perhaps we should just make `underlay` the default in the configs.

Reply:
There may be some for overlay when not running on a grid worker node.  I don't want to change the default from upstream, that's been a general principal.  We take the EPEL build unchanged.  The EPEL build had it off because the upstream singularity project didn't even support the feature and I wanted the default behavior to be the same as the upstream project.